### DISCUSSION TITLE
Audio works inconsistently (was issue 147)

### DISCUSSION THREAD

**Discussion by i0ntempest** [The-Wineskin-Project/WineskinServer#265] [Q&A] [answered]
Since the issue section is disabled for some reason in this repo I have to post here. I reported that in a free DX9 (Nitronic Rush) game the audio sometimes work and sometimes does not, and the issue is still present with the newest engine and wrapper today. Any news on this?

**Comment by Gcenx** [comment, marked as answer]
macOS natively has a different audio bitrate than Windows this causes these incompatibility’s.

**Reply by i0ntempest** [reply]
Any way I could change that either on the macOS side or the Wine side?

**Reply by Gcenx** [reply]
Some titles can simply be set to WinXP to avoid this, others need the audio device set to a different Hz I think the usual one is 96Hz.

This also affects Linux especially for something like Skyrim/Oblivian/NV where some users went an re-encoded the games audio to avoid the audio cracking/cutting out.

I’ve not tried Proton in years so not sure if there media-converter utility’s would help or not but considering wine32on64 currently doesn’t support gstreamer and I lightly can’t legally ship all gstreamer plugins there’s little I can really do on my end.

**Reply by i0ntempest** [reply]
My problem isn't audio cutting out or crackling, it works very well some times and other times completely muted.
Tried setting winxp in winetricks, didn't help. By setting audio device to 96kHz do you mean in macOS?
